src/packages/media/media/section-view/media-section-view.element.ts

```diff
@@ -1,48 +1,79 @@
 import { UMB_MEDIA_COLLECTION_ALIAS } from '../collection/index.js';
-import { css, html, customElement } from '@umbraco-cms/backoffice/external/lit';
+import type { UmbCollectionBulkActionPermissions, UmbCollectionConfiguration } from '../../../core/collection/types.js';
+import { css, html, customElement, state } from '@umbraco-cms/backoffice/external/lit';
 import { UmbLitElement } from '@umbraco-cms/backoffice/lit-element';
 import { UmbCollectionElement } from '@umbraco-cms/backoffice/collection';
+import { UmbDataTypeDetailRepository } from '@umbraco-cms/backoffice/data-type';
+import type { UmbDataTypeDetailModel } from '@umbraco-cms/backoffice/data-type';
 import type { UmbRoute } from '@umbraco-cms/backoffice/router';
+import { UmbPropertyEditorConfigCollection } from '@umbraco-cms/backoffice/property-editor';
 
 @customElement('umb-media-section-view')
 export class UmbMediaSectionViewElement extends UmbLitElement {
-	#routes: UmbRoute[] = [
-		{
-			path: 'collection',
-			component: () => {
-
-				// TODO: [LK] Work-in-progress. Need to get the data-type configuration for the Media Collection.
-				const config = {
-					unique: '',
-					dataTypeId: '', //'3a0156c4-3b8c-4803-bdc1-6871faa83fff', //'dt-collectionView',
-					allowedEntityBulkActions: {
-						allowBulkCopy: true,
-						allowBulkDelete: true,
-						allowBulkMove: true,
-						allowBulkPublish: false,
-						allowBulkUnpublish: false,
+	// TODO: [LK] Check if we can get the default data-type ID (for the Media ListView) from the server.
+	//private readonly defaultDataTypeId: string = '3a0156c4-3b8c-4803-bdc1-6871faa83fff';
+	private readonly defaultDataTypeUnique: string = 'dt-collectionView';
```

Comment:
Is this not the alias of our mock collection view? Would this work on the server?

Reply:
Cool 😎

Reply:
Yes, the old backoffice has the media section (dashboard) hardcoded for data-type ID `-96`.
https://github.com/umbraco/Umbraco-CMS/blob/release-13.1.1/src/Umbraco.Web.UI.Client/src/views/dashboard/dashboard.tabs.controller.js#L145

Which is the ID for the Media ListView data-type.
https://github.com/umbraco/Umbraco-CMS/blob/release-13.1.1/src/Umbraco.Core/Constants-DataTypes.cs#L31

So the question is, do we do the same and hardcode the data-type ID? 🤢 or do we ask the server for it? (but that means more work before the beta/rc) 🤔 

If we decide hardcode the ID for now, is it worth aligning the IDs from the server and mock data?
As on the server it is already hardcoded to `3A0156C4-3B8C-4803-BDC1-6871FAA83FFF`.
https://github.com/umbraco/Umbraco-CMS/blob/release-13.1.1/src/Umbraco.Core/Constants-DataTypes.cs#L115

Reply:
@kjac is aware of this problem. I think he knows the solution.

Reply:
Kenn says, as we also discussed on Slack, that you should just request the media collection with `null` as the `dataTypeId` and the server should be able to figure out which view is meant to be shown.